src/components/CustomerQuota/CustomerCard.tsx

```diff
@@ -76,11 +76,13 @@ export const AddButton: FunctionComponent<AddButton> = ({ text, onPress }) => {
   );
 };
 
-export const CustomerCard: FunctionComponent<{
-  ids: string[];
-  onAddId?: () => void;
-  headerBackgroundColor?: ViewStyle["backgroundColor"];
-}> = ({ ids, onAddId, headerBackgroundColor, children }) => {
+export const CustomerCard: FunctionComponent<
+  PropsWithChildren<{
```

Comment:
changes across 40 files but its all related to this change. previously `children` can be implicitly defined but now typescript wants us to explicitly define it